config/manifest/flux-installation-main.yaml

```
@@ -0,0 +1,51 @@
+---
+apiVersion: source.toolkit.fluxcd.io/v1
+kind: GitRepository
+metadata:
+  name: greenhouse-repo
+  namespace: default
+spec:
+  interval: 5m0s
+  timeout: 60s
+  url: https://github.com/cloudoperators/greenhouse
+  ref:
+    branch: main
+---
+apiVersion: kustomize.toolkit.fluxcd.io/v1
+kind: Kustomization
+metadata:
+  name: flux-kustomize
+  namespace: default
+spec:
+  interval: 10m
+  sourceRef:
+    kind: GitRepository
+    name: greenhouse-repo
+  path: "./config/flux/install"
```

This should point to the directory containing the kustomization.yaml?
```suggestion
  path: "./config/flux/"
```